### PR TITLE
Cleaned Up Core.ImageRecognitionFingerPrint, Core.GlobalAppInstances, Core.Logging

### PR DESCRIPTION
**Pull request by ob-ab**
- Renamed ImageRecognition.cs file to match Class name
- Cleaned up **ImageRecognitionFingerPrint.cs** file
- Cleaned up **GlobalAppInstances.cs** file
- Cleaned up **Logging.cs** file
- Updated dependent file(s)

### CODE REVIEW

File: taskt/Core/ImageRecognitionFingerPrint.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
+﻿using System.Drawing;
+
+namespace taskt.Core
+{
+    /// <summary>
+    /// Used by Image Recognition to generate and verify a unique fingerprint of an image
+    /// </summary>
+    public class ImageRecognitionFingerPrint
+    {
+        public int PixelId { get; set; }
```
Comment (openbots-ff) [inline taskt/Core/ImageRecognitionFingerPrint.cs]: I feel like this file might be in the wrong place. It feels kind of random for the Core folder. What do you think?

**Review by openbots-ff — CHANGES_REQUESTED** [review]
Looks good. Just let me know what you think of the comment I left

File: taskt/Core/ImageRecognitionFingerPrint.cs (hunk 1 above)
Comment (ob-ab) [reply taskt/Core/ImageRecognitionFingerPrint.cs]: I know, let's discuss its placement in Standup.

**Review by openbots-ff — DISMISSED** [review]

File: taskt/Core/Automation/Commands/Image Commands/ImageRecognitionCommand.cs [hunk 2]
```diff
@@ -6,6 +6,7 @@
 using System.Windows.Forms;
 using taskt.UI.Forms;
 using taskt.UI.CustomControls;
+using taskt.Core.Utilities.CommandUtilities;
 
 namespace taskt.Core.Automation.Commands
```
Comment (CameronHerwig) [inline taskt/Core/Automation/Commands/Image Commands/ImageRecognitionCommand.cs:11]: Should review method in refactor pass.

File: taskt/Core/Automation/Engine/AutomationEngineInstance.cs [hunk 3]
```diff
@@ -7,6 +7,8 @@
 using System.Linq;
 using System.Threading;
 using taskt.Core.Server;
+using taskt.Core.Utilities.CommonUtilities;
+using taskt.Core.App;
 using taskt.UI.Forms;
 using taskt.Core.Script;
 using taskt.Core.Automation.Commands;
```
Comment (CameronHerwig) [inline taskt/Core/Automation/Engine/AutomationEngineInstance.cs:14]: Are all of these references utilized?

File: taskt/Core/Utilities/CommonUtilities/Logging.cs [hunk 4]
```diff
@@ -0,0 +1,33 @@
+﻿using Serilog;
+using Serilog.Core;
+using Serilog.Formatting.Compact;
+using taskt.Core.IO;
+
+namespace taskt.Core.Utilities.CommonUtilities
```
Comment (CameronHerwig) [inline taskt/Core/Utilities/CommonUtilities/Logging.cs:6]: Will need to do a review as part of Server execution log integration.

**Review by CameronHerwig — DISMISSED** [review]

File: taskt/Core/Automation/Engine/AutomationEngineInstance.cs (hunk 3 above)
Comment (ob-ab) [reply taskt/Core/Automation/Engine/AutomationEngineInstance.cs:14]: Yes

File: taskt/Core/ImageRecognitionFingerPrint.cs (hunk 1 above)
Comment (ob-ab) [reply taskt/Core/ImageRecognitionFingerPrint.cs]: Moved to Core/Utilities/CommandUtilities Directory

**Review by CameronHerwig — APPROVED** [review]

**Review by openbots-ff — APPROVED** [review]

**Pull request merged**